src/ai/backend/manager/registry.py

```diff
@@ -3149,19 +3151,21 @@ async def _update() -> None:
                 )
 
             # Update the mapping of kernel images to agents.
-            loaded_images = msgpack.unpackb(zlib.decompress(agent_info["images"]))
+            images = msgpack.unpackb(zlib.decompress(agent_info["images"]))
+            image_canonicals = set(img_info[0] for img_info in images)
```

Comment:
I assume the 0th value means canonical, but that's not going to change, right?

Reply:
Yes, FYI,
Tuple's 0th value: image canonical.
Tuple's 1st value: image digest.